flink-libraries/flink-table/src/main/scala/org/apache/flink/api/table/runtime/aggregate/AggregateMapFunction.scala

```diff
@@ -30,9 +30,9 @@ class AggregateMapFunction[IN, OUT](
     private val groupingKeys: Array[Int],
     @transient private val returnType: TypeInformation[OUT])
     extends RichMapFunction[IN, OUT] with ResultTypeQueryable[OUT] {
-  
```

Comment:
yes,i will do it .
By the way Can we declare the style of the uniform Class / Object / Trait / Method as shown in the following example?

1. Class/Object/Trait put each constructor argument on its own line, indented four spaces:
If a class/object/trait extends anything, put each constructor argument on its own line, indented four spaces and two spaces for extensions:

// wrong!
'''
class DataStreamAggregate(
  window: LogicalWindow,
  namedProperties: Seq[NamedWindowProperty],
  cluster: RelOptCluster,
  traitSet: RelTraitSet,
  inputNode: RelNode,
  namedAggregates: Seq[CalcitePair[AggregateCall, String]],
  rowRelDataType: RelDataType,
  inputType: RelDataType,
  grouping: Array[Int])
  extends SingleRel(cluster, traitSet, inputNode)
          with FlinkAggregate
          with DataStreamRel {
'''

// right!
'''
class DataStreamAggregate(
    window: LogicalWindow,
    namedProperties: Seq[NamedWindowProperty],
    cluster: RelOptCluster,
    traitSet: RelTraitSet,
    inputNode: RelNode,
    namedAggregates: Seq[CalcitePair[AggregateCall, String]],
    rowRelDataType: RelDataType,
    inputType: RelDataType,
    grouping: Array[Int])
  extends SingleRel(cluster, traitSet, inputNode)
  with FlinkAggregate
  with DataStreamRel {
  '''

2. Methods with Numerous Arguments  indented two spaces from the current indent level:

// wrong!
'''private[flink] def createIncrementalAggregateReduceFunction(
       aggregates: Array[Aggregate[_ <: Any]],
       namedAggregates: Seq[CalcitePair[AggregateCall, String]],
       inputType: RelDataType,
       outputType: RelDataType,
       groupings: Array[Int]): IncrementalAggregateReduceFunction = {
 '''

// right!
 '''
 private[flink] def createIncrementalAggregateReduceFunction(
   aggregates: Array[Aggregate[_ <: Any]],
   namedAggregates: Seq[CalcitePair[AggregateCall, String]],
   inputType: RelDataType,
   outputType: RelDataType,
   groupings: Array[Int]): IncrementalAggregateReduceFunction = {

 '''